<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IR_Plotter</class>
 <widget class="QWidget" name="IR_Plotter">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1270</width>
    <height>739</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IR Spectrum Processor</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>1261</width>
     <height>721</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="3">
     <widget class="QPushButton" name="uiShowGridButton">
      <property name="text">
       <string>Show grid</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QSpinBox" name="uiOrderBandsSpinBox">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="value">
       <number>20</number>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="uiBandManualButton">
      <property name="text">
       <string>Band Picking (Manual)</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="9">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="10">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Bands Report</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QLabel" name="label_3">
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Bands order:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="label_4">
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Bands threshold:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Baseline threshold:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <widget class="QPushButton" name="uiRestoreButton">
      <property name="text">
       <string>Restore</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="uiPercentTransButton">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>%Transmittance</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="9">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="5">
     <widget class="QPushButton" name="uiShowLabelsButton">
      <property name="text">
       <string>Show/Hide labels</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="uiSubsBaselineButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Substract Baseline</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="6">
     <layout class="QVBoxLayout" name="uiToolbarLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
     </layout>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="uiBandAutoButton">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Band Picking (Auto)</string>
      </property>
     </widget>
    </item>
    <item row="5" column="10">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Save Report</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" rowspan="2" colspan="10">
     <layout class="QVBoxLayout" name="uiSpectraLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
     </layout>
    </item>
    <item row="5" column="3">
     <widget class="QDoubleSpinBox" name="uiBandsThresholdSpinBox">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.000100000000000</double>
      </property>
      <property name="value">
       <double>0.997500000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="uiAbsorbanceButton">
      <property name="text">
       <string>Absorbance</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QLabel" name="label_2">
      <property name="maximumSize">
       <size>
        <width>110</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Baseline order:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="uiShowBandsButton">
      <property name="text">
       <string>Show/Hide bands</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QSpinBox" name="uiOrderBaseSpinBox">
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="value">
       <number>70</number>
      </property>
     </widget>
    </item>
    <item row="2" column="10" rowspan="3">
     <widget class="QTableView" name="uiIRBandsTableView"/>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="uiDetectBaselineButton">
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Detect Baseline</string>
      </property>
     </widget>
    </item>
    <item row="1" column="8" colspan="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="3">
     <widget class="QDoubleSpinBox" name="uiBaseThresholdSpinBox">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.000100000000000</double>
      </property>
      <property name="value">
       <double>0.997500000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="uiTransButton">
      <property name="text">
       <string>Transmittance</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="7" colspan="2">
     <widget class="QPushButton" name="uiBandFittingAutoButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Band Fitting (Auto)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QPushButton" name="uiNormalizeButton">
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Normalize</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
